src/RemoteInboxNotifications/RemoteInboxNotificationsEngine.php

```diff
@@ -73,12 +76,21 @@ public static function on_admin_init() {
 
 		add_action( 'activated_plugin', array( __CLASS__, 'run' ) );
```

Comment:
I wonder if it's worth considering making the engine run here async as well?

If this is agreed, I would also suggest adding the `woocommerce_activated_plugin` action to cater for [wcadmin embedded](https://github.com/woocommerce/woocommerce/pull/28698).

Reply:
This is probably a good idea, I can't see a reason not to make it async apart from introducing complexity when there isn't a huge amount of gain - plugins would generally be activated much less than products are added/edited, and there isn't much much overhead in running the engine as it doesn't actually pull down from the feeds except on a first run when WCA is installed.

The extra action makes sense too. I'll create these as a follow-up ticket though.

Reply:
I agree, I think the added complexity might not worth the performance add. Thanks for making the follow-up ticket for the added action!